docs/msbuild/updating-an-existing-application.md

```diff
@@ -44,6 +44,7 @@ For example, you can use this XML:
 <ItemGroup>
   <PackageReference Include="Microsoft.Build" Version="15.1.548" ExcludeAssets="runtime" />
   <PackageReference Include="Microsoft.Build.Utilities" Version="15.1.548" ExcludeAssets="runtime" />
+  <PackageReference Include="Microsoft.Build.Locator" Version="1.1.2"  />
```

Comment:
This is specified separately in the `Add package reference` section below. For step-by-step upgrade instructions, I think it makes sense to do the upgrade in two phases:

* Move to package references
* Move to Locator

But this is definitely confusing if you just come to this document looking for "what does this look like in the end?" which is a totally reasonable thing to do. @Mikejo5000 should we have a separate doc for that use case and establish some cross-links?

Reply:
@Mikejo5000 can you take a look here and recommend the next step?

Reply:
@rainersigwald Rainer, the new non-sequential use case here would be "How to reference MSbuild"? Which would apply equally to MSBuild 15.0 and 16.0.